src/patterns/components/pagination/angular/code/long.hbs

```diff
@@ -0,0 +1,20 @@
+<sprk-pagination
+  additionalClasses="sprk-u-mbl"
+  paginationType="long"
+  nextLinkText="Next"
+  prevLinkText="Prev"
+  [currentPage]="longConfig[0].currentPage"
+  [totalItems]="longConfig[0].totalItems"
+  [itemsPerPage]="longConfig[0].itemsPerPage"
+  (previousClick)="yourGoBackFunction($event)"
+  (pageClick)="yourGoToPageFunction($event)"
+  (nextClick)="yourGoForwardFunction($event)">
+</sprk-pagination>
+
+longConfig = [
```

Comment:
this might be confusing, people might think there's like a config input

Reply:
Ahh yeah, I can see that, I'll remove it. Good catch!